tests/tasks/jupyter/test_jupyter.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+
+from prefect.tasks.jupyter import JupyterTask
+
+pytest.importorskip("papermill")
+
+
+def test_jupyter():
+    task = JupyterTask(
+        "tests/tasks/jupyter/sample_notebook.ipynb",
```

Comment:
Did you leave this file out? I think I'd prefer if you used the pytest `tmpdir` fixture and wrote this file to disk for the test.

Reply:
Not sure how to use the `tmpdir` fixture for this. Can you point me to an example? And where should I put the file if I do that

Reply:
Now that you've attached the file, it's small enough that I don't mind it being in the repo although it's possible someone else will have a different opinion. Unless someone else thinks this would be better, don't worry about it. Anyway, here's an example of the tmpdir fixture for future reference: https://docs.pytest.org/en/stable/tmpdir.html#the-tmpdir-fixture

Reply:
The fixture is cool, thanks!
Nevertheless, I'd still need a place to put the actual data, and then load it into the fixture, wouldn't I?

Reply:
You'd put it in a string literal in the test which would make it faster for someone to see what's being run in the notebook. That said, the file is a little long for that so your solution is probably ideal :) There's probably  a function (in jupyter?) that generates most of the notebook metadata for you so it'd be more concise but I don't see a reason to try to track that down.